src/index.html

```diff
@@ -5,8 +5,452 @@
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymus"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="logo"
+          href="#home"
+        >
+          <img
+            src="/images/logo.png"
+            alt="MOYO logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#apple" class="nav__link nav__link--active is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="noutes"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="gadzety" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#planshety" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog main__catalog">
+        <div class="catalog__container">
+          <div class="card catalog__card" data-qa="card">
+            <img
+             class="card__product-image"
+             src="./images/imac.jpeg"
+              alt="imac"
+            >
+
+            <div class="card__discription">
+              Моноблок APPLE A1419 iMac 27&#34; Retina 5K (MNED2UA/A)
+            </div>
+
+            <div class="card__code">
+              Код товара: 195434
+            </div>
+
+            <div class="card__feedback">
+              <div class="stars stars--raiting_4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__responses">
+                Отзывов: 5
+              </div>
+            </div>
+
+            <div class="card__price">
+              <div class="card__price-title">
+                  Цена:
+              </div>
+
+              <div class="card__price-amount">
+                  69 999 грн
+              </div>
```

Comment:
The same here. [Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 1st point